daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -365,17 +371,10 @@ object Node {
   }
 
   final case class NodeRollback[+Nid](
-      // TODO https://github.com/digital-asset/daml/issues/8020
-      // Figure-out what information needs to be contained in a Rollback node.
-      // For the moment, we just have the children.
-      // Note: if we add values, we must extend the function which checks they are serializable
-      children: ImmArray[Nid],
-      // For the sake of consistency between types with a version field, keep this field the last.
-      override val version: TransactionVersion,
+      children: ImmArray[Nid]
   ) extends GenNode[Nid, Nothing] {
 
-    override private[lf] def updateVersion(version: TransactionVersion): NodeRollback[Nid] =
-      copy(version = version)
+    private[lf] def updateVersion(version: TransactionVersion): NodeRollback[Nid] = this
```

Comment:
Similar to above, this seems convenient but really updating the version of a rollback node doesn’t make sense so maybe only having this method on action nodes is preferable?

Reply:
replaced this method with a function at the call site